source/_footer.erb

```diff
@@ -0,0 +1,67 @@
+<div class="footer">
+  <div class="footer-container">
+    <div class="footer-container__address-tile">
+      <p class="footer-container__company-name">
+        unboxed
+      </p>
+      <p>hello@unboxed.co</p>
+      <div class="social-icons">
+        <a class="social-icons__icon-container" href="http://twitter.com/Ubxd" target="_blank">
+          <div class="social-icons__icon--twitter">
+            <%= image_tag 'twitter.png', class: 'social-icons__image', alt: 'Twitter' %>
+          </div>
+        </a>
+        <a class="social-icons__icon-container" href="http://www.linkedin.com/company/unboxed-consulting" target="_blank">
+          <div class="social-icons__icon--linkedin">
+            <%= image_tag 'linkedin.png', class: 'social-icons__image', alt: 'LinkedIn' %>
+          </div>
+        </a>
+      </div>
+      <div class="footer-container__registration-address">
+        <p>registered in England and Wales</p>
+        <p>05440833</p>
+      </div>
+    </div>
+    <div class="footer-container__address-tile">
+      <p class="footer-container__address-city"><a href="http://goo.gl/maps/AuKrs" target="_blank"><i class="fa fa-map-marker footer__map-icon"></i>
+        London
+      <span>office</span></a></p>
+      <a class="footer-container__address-phone" href="tel:+442071834250"><i class="fa fa-phone footer__phone-icon"></i>
+        +44 20 7183 4250
+      </a>
+      <div class="footer-container__address-full">
+        <p>17 Blossom Street</p>
+        <p>London, E1 6PL</p>
+        <p>United Kingdom</p>
+      </div>
+    </div>
+    <div class="footer-container__address-tile">
+      <p class="footer-container__address-city"><a href="http://goo.gl/maps/JkNC9" target="_blank"><i class="fa fa-map-marker footer__map-icon"></i>
+        Cape Town
+      <span>office</span></a></p>
+      <a class="footer-container__address-phone" href="tel:+27216714503"><i class="fa fa-phone footer__phone-icon"></i>
+        +27 21 671 4503
+      </a>
+    <div class="footer-container__address-full">
+      <p>303 Buchanan Chambers,</p>
+      <p>Cnr Pearce & Warwick Street</p>
+      <p>Claremont, 7735</p>
+      <p>South Africa</p>
+     </div>
+    </div>
+    <div class="footer-container__newsletter">
+       <p>Sign up to unboxed updates</p>
```

Comment:
Capital U? I wonder whether we are "unboxed" or "Unboxed"... I would have thought the latter, but it's only the logo is lowercase.